distribution/lib/Standard/Base/0.0.0-dev/src/Data/List.enso

```diff
@@ -209,9 +209,18 @@ type List
     map self f = case self of
         Nil -> Nil
         Cons h t ->
-            res = Cons (f h) Nil
-            map_helper t res f
-            res
+            go : List -> Any -> (Any -> Any) -> Nothing
+            go list fill = case list of
+                Nil -> fill Nil
+                Cons h t ->
+                    v = f h
+                    res = Meta.atom_with_hole (e -> Cons v e)
```

Comment:
```suggestion
                    res = Meta.atom_with_hole (Cons v _)
```
Think that might read better (and I believe equivalent)

Reply:
Thanks for explaining me the meaning of `_`. Used since a8cd2bf6bed3a60cc9d94863663558943e72e0eb